### PR TITLE
Installation

### PR DESCRIPTION
**Pull request by brianteeman**
As discussed with @mbabker we can permanently resolve the issue of the name of the installation folder from being translated by removing it from the string and inserting it dynamically using JText::sprintf

All other attempts have failed so this is the only way we can ensure that users are presented with the correct information

Updated PR to work around some guthub issue with #17276

### CODE REVIEW

**Comment by mbabker** [comment]
For whatever reason it was still missing that file's changes here, I just fixed it.  We should be good now.

File: installation/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -122,20 +122,14 @@ INSTL_EMAIL_NOT_SENT="Email could not be sent."
...
 INSTL_COMPLETE_LANGUAGE_DESC2="Note: you will need internet access for Joomla! to download and install the new languages. <br />Some server configurations won't allow Joomla! to install the languages. If this is your case, don't worry, you will be able to install them later using the Joomla! Administrator."
-; The word 'installation' should not be translated as it is a physical folder.
-INSTL_COMPLETE_REMOVE_FOLDER="Remove installation folder"
-; The word 'installation' should not be translated as it is a physical folder.
-INSTL_COMPLETE_REMOVE_INSTALLATION="PLEASE REMEMBER TO COMPLETELY REMOVE THE INSTALLATION FOLDER.<br />You will not be able to proceed beyond this point until the installation folder has been removed. This is a security feature of Joomla!"
+INSTL_COMPLETE_REMOVE_FOLDER="Remove &quot;%s&quot; folder"
+INSTL_COMPLETE_REMOVE_INSTALLATION="PLEASE REMEMBER TO COMPLETELY REMOVE THE INSTALLATION FOLDER.<br />You will not be able to proceed beyond this point until the &quot;%s&quot; folder has been removed. This is a security feature of Joomla!"
```
Comment (Quy) [inline installation/language/en-GB/en-GB.ini:132]: Is this one ok? `REMOVE THE INSTALLATION FOLDER.`

Comment (brianteeman) [reply installation/language/en-GB/en-GB.ini:132]: yes i think so as it is displayed as a title

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9c985e875a641c9aca6b30e4a5dad11e94e1391c<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17436">issues.joomla.org/tracker/joomla-cms/17436</a>.</sub>

**Comment by brianteeman** [comment]
Thanks for merging

**Comment by zero-24** [comment]
why did we merge that with a red drone: http://213.160.72.75/joomla/joomla-cms/7860 ? 

image

I'm going to prepare a PR..

**Comment by mbabker** [comment]
Missed that, my bad.

**Comment by zero-24** [comment]
here it is https://github.com/joomla/joomla-cms/pull/17544

**Pull request merged**